Gopkg.toml

```diff
@@ -1,14 +1,14 @@
 [[constraint]]
   name = "github.com/DataDog/datadog-agent"
-  revision = "4aa90bd394b62b0d3e3f15703c6e459c1c477fbc"
+  revision = "87fcbdb53630c978b8561d2c2272b285123c3d42"
 
 [[constraint]]
   name = "github.com/DataDog/gopsutil"
   revision = "c8f74f1344dd41bc49ec4d3c2377c526aaedfd20"
 
 [[constraint]]
   name = "github.com/gogo/protobuf"
-  revision = "d76fbc1373015ced59b43ac267f28d546b955683"
+  version = "~1.0.0"
```

Comment:
This is required due to a change in `datadog-agent`. We'll have to test this well.

Reply:
Any way to pin this to a hard version?

Reply:
I'd like to keep it in-sync with `datadog-agent`: 
     https://github.com/DataDog/datadog-agent/blob/master/Gopkg.toml#L41